vllm/model_executor/models/deepseek_v2.py

```diff
@@ -555,9 +564,16 @@ def forward(
         )
 
         # Fully Connected
+        if isinstance(self.mlp, DeepseekV2MoE) and \
```

Comment:
ditto on the comment. This will keep us sane in the future :)

Reply:
Comment added